unity/unity-embed-host/CoreCLRHost.cs

```diff
@@ -671,6 +671,15 @@ public static IntPtr unity_method_get_function_pointer([NativeCallbackType("Mono
         return metBase.IsGenericMethodDefinition;
     }
 
+    [NativeFunction(nameof(unity_mono_method_is_inflated))]
+    [return: NativeCallbackType("gboolean")]
+    public static bool unity_mono_method_is_inflated(
+        [NativeCallbackType("MonoMethod*")] IntPtr method)
+    {
+        MethodBase metBase = MethodBase.GetMethodFromHandle(method.MethodHandleFromHandleIntPtr());
+        return !metBase.IsGenericMethodDefinition && metBase.IsGenericMethod;
```

Comment:
I'm pretty sure that a non-generic method on a generic type should also return `true` for is inflated.  e.g.  `GenericAnimal<int, int>::InterfaceMethodOnIGenericAnimal` should return true.   We'll need to double check that.

Reply:
I am getting false as the method is not generic, but am unsure if that is the expected result

Reply:
Checked on current implementation, a non-generic method inside an inflated class is indeed considered inflated.
Will fix the PR accordingly.